project/MimaExcludes.scala

```diff
@@ -60,6 +60,10 @@ object MimaExcludes {
               "org.apache.spark.ml.regression.LeastSquaresCostFun.this"),
             ProblemFilters.exclude[MissingMethodProblem](
               "org.apache.spark.ml.classification.LogisticCostFun.this"),
+            ProblemFilters.exclude[MissingMethodProblem](
```

Comment:
Can we preserve binary compatibility by providing an overload with the original signature?